src/rgw/driver/dbstore/common/dbstore.cc

```diff
@@ -2063,7 +2051,7 @@ int DB::set_entry(const std::string& oid, rgw::sal::Lifecycle::LCEntry& entry)
 }
 
 int DB::list_entries(const std::string& oid, const std::string& marker,
-  				 uint32_t max_entries, std::vector<std::unique_ptr<rgw::sal::Lifecycle::LCEntry>>& entries)
+                     uint32_t max_entries, std::vector<rgw::sal::LCEntry>& entries)
```

Comment:
std::vector of entries (expanded)?

Reply:
right, this just unwraps the individual entries from unique pointers